### PR TITLE
PAYARA-1618 outputuberjar packages up applications deployed to rootdir

### PR DESCRIPTION
**Pull request by Cousjava**

### CODE REVIEW

**Comment by Cousjava** [comment]
jenkins test please

File: appserver/extras/payara-micro/payara-micro-core/src/main/java/fish/payara/micro/impl/UberJarCreator.java [hunk 1]
```diff
@@ -338,6 +341,28 @@ public void buildUberJar() {
                     }
 
                 }
+                
+                File applicationsDir = new File(domainDir, "applications");
+                if (applicationsDir.exists()){
+                    for (File app : fillFiles(applicationsDir)){
+                        String path = app.getPath();
+                        if (path.endsWith(".war") || path.endsWith(".jar") || path.endsWith(".rar")){
```
Comment (smillidge) [inline appserver/extras/payara-micro/payara-micro-core/src/main/java/fish/payara/micro/impl/UberJarCreator.java]: Also needs .ear

File: appserver/extras/payara-micro/payara-micro-core/src/main/java/fish/payara/micro/impl/UberJarCreator.java [hunk 2]
```diff
@@ -338,6 +341,28 @@ public void buildUberJar() {
...
+                            JarEntry appEntry = new JarEntry("MICRO-INF/deploy/" + app.getName());
+                            appEntry.setMethod(JarEntry.STORED);
```
Comment (smillidge) [inline appserver/extras/payara-micro/payara-micro-core/src/main/java/fish/payara/micro/impl/UberJarCreator.java]: deployments don't need to be in STORED format

**Comment by Cousjava** [comment]
jenkins test please

**Pull request merged**